README.md

```diff
@@ -14,6 +14,8 @@ Python is a dev dependency. It's not needed to run the docker containers, but ne
   One or both of the following, depending on if you want to run the stable build `latest` or `dev`:
     - a. `docker pull obolibrary/odkfull:latest`
     - b. `docker pull obolibrary/odkfull:dev`
+4. Optional: [`odkrunner`](https://github.com/gouttegd/odkrunner): Alternative to `run.sh`. If using this, you would run
+    like `odkrun make build-mondo-ingest` instead of `sh run.sh make build-mondo-ingest`.
```

Comment:
Why not simply odk, instead of odkrun, that's what I do.

Reply:
I was thinking that too when I first saw this.

Well, we can do that and update these docs to something like:
> Suggestion: Rename the binary to `odk`, and use like `odk make build-mondo-ingest` instead of `sh run.sh make build-mondo-ingest`

What do you think?

Reply:
That's at least what I do

Reply:
Oh, lol, I had a conflict with an existing `odk` on my machine: [Open Data Kit](https://getodk.org/)

I was much more intimately involved with that ODK than Ontology Development Kit, actually, lol. But no more.